packages/cloud-cognitive/src/components/Datagrid/Extensions/RowHeightSettings/RowHeightSettings.stories.js

```diff
@@ -151,7 +155,7 @@ const BasicUsage = ({ ...args }) => {
       {
         Header: 'Someone 11',
         accessor: 'someone11',
-        multiLineWrap: true,
+        multiLineWrap: false,
```

Comment:
We can probably remove this whole section if we aren’t trying to show multiline in this story anymore.